sphinxcontrib/needs/warnings.py

```diff
@@ -50,10 +52,17 @@ def process_warnings(app, exception):
                 logger.info("  {}: passed".format(warning_name))
             else:
                 need_ids = [x["id"] for x in result]
-                logger.info("  {}: failed".format(warning_name))
-                logger.info("  \t\tfailed needs: {} ({})".format(len(need_ids), ", ".join(need_ids)))
-                logger.info("  \t\tused filter: {}".format(warning_filter))
-                warning_raised = True
+                if warnings_always_warn:
+                    logger.warning(
+                        "  {}: failed\n  \t\tfailed needs: {} ({})\n  \t\tused filter: {}".format(
+                            warning_name, len(need_ids), ", ".join(need_ids), warning_filter
+                        )
+                    )
+                else:
+                    logger.info("  {}: failed".format(warning_name))
```

Comment:
Please use here the same code for logging as above

Reply:
done.